### PR TITLE
fix: remove polluting google log library

### PR DESCRIPTION
**Pull request by aeneasr**
For more context please see discussion https://discuss.dgraph.io/t/ristretto-0-1-0-pollutes-the-flags-namespace/14561

By reverting #263 not only is the number of dependencies this library has greatly reduced, but projects using this library will not unavoidably panic.

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/ristretto/292)

### CODE REVIEW

File: contrib/demo/node_allocator.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+//go:build jemalloc && allocator
```
Comment (aeneasr) [inline contrib/demo/node_allocator.go:1]: Done by Go 1.17 `gofmt`

**Comment by floatdrop** [comment]
@manishrjain hey, can this be merged? glog pollutes our binary flags. Thanks!

**Comment by manishrjain** [comment]
Hey @floatdrop ,

I'm no longer associated with Dgraph Labs. I'm not maintaining repositories owned by dgraph-io. I've forks of these repos in outcaste-io. If you want to switch to that and send me the PR, I'll be happy to review it.

https://github.com/outcaste-io/ristretto

**Comment by aeneasr** [comment]
Is anyone maintaing this repo or is it essentially abandoned?

**Comment by karlmcguire** [comment]
@aeneasr Abandoned, check out https://github.com/outcaste-io/ristretto

**Comment by mangalaman93** [comment]
A similar PR is merged now https://github.com/dgraph-io/ristretto/pull/350

**Pull request closed without merging**